tests/functional/image-uploads-non-auth.js

```diff
@@ -168,7 +168,7 @@ registerSuite("Image Uploads (non-auth)", {
           .execute(() => {
             // work around for chrome "Other element would receive the click"
             // error
-            $(".js-remove-upload")[0].click();
+            document.querySelector(".js-remove-upload").click();
```

Comment:
@miketaylr you can define jquery globally in webpack configuration.

Reply:
Yeah, it's no longer available globally on the window object (e.g. window.$ or window.Jquery), so I decided to use DOM methods instead. 

However jst templates still use jquery and underscore without explicitly importing it, so I made it "global" for the bundle for now with https://webpack.js.org/plugins/provide-plugin/#usage-jquery, which exposes a corresponding module once "_" or "$" is found in the code with no explicit import
https://github.com/webcompat/webcompat.com/blob/4bb999e2c08cd9629723912576e7174ed2c90dfb/webpack/webpack.common.js#L58-L65

It's also possible to make jquery available on the window object with this plugin as @magsout mentioned, but I don't think we need it, since it would be beneficial only for tests 
```js
new webpack.ProvidePlugin({
  'window.jQuery': 'jquery'
});
```

Reply:
> @miketaylr you can define jquery globally in webpack configuration.

Cool good to know.

> I don't think we need it

Yeah, it's overkill for just selecting a DOM element.